### PR TITLE
Hauki 83 react axe

### PR DESCRIPTION
**Pull request by VilleMiekkoja**

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // ***********************************************************
```
Comment (minevala) [inline cypress/support/index.js:1]: Would you change the extensions of the other files from .js to .ts too?

File: cypress/integration/App.spec.ts [hunk 2]
```diff
@@ -1,6 +1,14 @@
+/// <reference types="cypress-axe" />
+
 describe('Open aukiolot app', () => {
```
Comment (minevala) [inline cypress/integration/App.spec.ts:3]: I get the following lint warning about this in my IDE: 
`TS1208: All files must be modules when the '--isolatedModules' flag is provide`

**Review by minevala — COMMENTED** [review]
I commented on a couple of small things. It is nice that we also have an automated way to test accessibility 👍

File: cypress/integration/App.spec.ts (hunk 2 above)
Comment (minevala) [reply cypress/integration/App.spec.ts:3]: Näyttökuva 2020-9-23 kello 17 23 55

Comment (VilleMiekkoja) [reply cypress/integration/App.spec.ts:3]: This should be fixed now

**Review by minevala — APPROVED** [review]
🥇

**Pull request merged**